CMakeLists.txt

```diff
@@ -48,6 +48,16 @@ set(TC_PLATFORM_WIN32 false)
 if (APPLE)
   set(TC_PLATFORM_APPLE true)
   add_definitions(-DTC_PLATFORM_APPLE)
+  find_library(ACCELERATE NAMES Accelerate)
+  find_library(APPKIT AppKit)
+  find_library(CORE_GRAPHICS CoreGraphics)
+  find_library(JAVASCRIPTCORE JavaScriptCore)
```

Comment:
Okay, I'm going to be that guy. Shouldn't this be `JAVA_SCRIPT_CORE`?

Reply:
Actually I think it should be JAVASCRIPT_CORE (since JavaScript is all one word).